src/sentry/incidents/tasks.py

```diff
@@ -140,3 +142,41 @@ def handle_trigger_action(action_id, incident_id, project_id, method):
         )
     )
     getattr(action, method)(incident, project)
+
+
+@instrumented_task(
+    name="sentry.incidents.tasks.auto_resolve_snapshot_incidents",
+    queue="incidents",
+    default_retry_delay=60 * 5,
+    max_retries=1,
+)
+@retry(exclude=(DeleteAborted,))
```

Comment:
I'd probably avoid using `DeleteAborted` here, since it's specific to our deletion jobs. You could define your own exception instead if you want a specific thing.